### PR TITLE
Governance file

### PR DESCRIPTION
**Pull request by arcanis**
**What's the problem this PR addresses?**
The governance isn't transparent. I think we all have a fairly good idea what it is, but it can be worth putting it in words - especially if we're looking at accepting sponsorships in the future.

**How did you fix it?**
I started from the [Rome](https://github.com/rome/tools/blob/main/GOVERNANCE.md) governance model, since it presents many similarities with the situation here, and slightly adapted it.

cc @yarnpkg/berry

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+## Moderation
+
+Outlined below is the process for Code of Conduct violation reviews.
+
+### Reporting
+
+Anyone may report a violation. Violations can be reported in the following ways:
+
+- In private, via email to one or more stewards
+- In private, via direct message to a project steward on Discor
```
Comment (merceyz) [inline GOVERNANCE.md:92]:
```suggestion
- In private, via direct message to a project steward on Discord
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+- Kristoffer K. - [merceyz](https://github.com/merceyz)
+- Marc-Antoine - [embraser01](https://github.com/embraser01)
+- Paul Soporan - [paul-soporan](https://github.com/paul-soporan)
+- Sebastian Silbermann - [eps1lon](https://github.com/eps1lon)
+- Victor Vlasenko - [larixer](https://github.com/larixer)
+- Will Griffiths - [willgriffiths](https://github.com/willgriffiths)
+
+## Project direction and planning
+
+Project direction and planning is a shared responsibility amongst members. Stewards are responsible for dictating high level goals and scope of the project that should be adhered to.
```
Comment (eps1lon) [inline GOVERNANCE.md:64]:
```suggestion
Project direction and planning is a shared responsibility amongst members. Stewards are responsible for defining high level goals and scope of the project that should be adhered to.
```

"Dictating" sounds like you're telling people what they should do and has, for me personally, a rather negative connotation.

**Review by eps1lon — APPROVED** [review]
Thanks for putting this together. Really appreciate it :+1:

File: GOVERNANCE.md (hunk 2 above)
Comment (arcanis) [reply GOVERNANCE.md]: Much better indeed 👍

**Review by merceyz — APPROVED** [review]

**Review by paul-soporan — APPROVED** [review]

**Review by larixer — APPROVED** [review]

**Review by bgotink — APPROVED** [review]

**Review by deini — APPROVED** [review]

**Review by Haroenv — APPROVED** [review]

**Review by Embraser01 — APPROVED** [review]

**Pull request merged**